lib/index.js

```diff
@@ -161,16 +169,16 @@ internals.Client.prototype._request = function (method, url, options, relay, _tr
     _trace = (_trace || []);
     _trace.push({ method: uri.method, url });
 
-    const client = (uri.protocol === 'https:' ? Https : Http);
+    const client = (parsedUri.protocol === 'https:' ? Https : Http);
```

Comment:
It looks like the `applyUrlToOptions` will set `protocol` on the `uri`, therefore do we need to change this to `parsedUri`?

Reply:
Oh, you're right. This was a holdout from my cleanup after adding `applyUrlToOptions` . I'll fix that :+1: